views/components/ChartList.jsx

```diff
@@ -29,39 +30,48 @@ const ChartList = () => {
         })
       }
     } catch (error) {
-      console.error(error, "*****")
+      console.error(error, '*****')
     }
   }
+  const editChart = async (id) => window.location.assign(routes.editChart(id))
 
   return (
     <Table>
       <TableHead>
         <TableRow>
-          <TableCell align="center">Title</TableCell>
-            <TableCell align="center">Description</TableCell>
-            <TableCell align="center">Delete</TableCell>
-          </TableRow>
+          <TableCell align='center'>Title</TableCell>
+          <TableCell align='center'>Description</TableCell>
+          <TableCell align='center'>Edit</TableCell>
+          <TableCell align='center'>Delete</TableCell>
+        </TableRow>
       </TableHead>
       <TableBody>
-        {chartList.map(({title, description, _id}) => (
+        {chartList.map(({ title, description, _id }) => (
           <TableRow key={_id}>
-            <TableCell align="center">
+            <TableCell align='center'>
               <Button
-                type="button"
-                variant="text"
+                type='button'
+                variant='text'
                 fullWidth
                 href={routes.chart(_id)}
               >
                 {title?.toUpperCase()}
               </Button>
             </TableCell>
-            <TableCell align="center">
-              {description?.toUpperCase()}
+            <TableCell align='center'>{description?.toUpperCase()}</TableCell>
+            <TableCell align='center'>
+              <Button
+                type='button'
+                variant='text'
+                onClick={() => editChart(_id)}
```

Comment:
can this be a link instead of a button (it can still look like a button) so we can pass an href here and not need to handle navigating using window.location?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870